### ISSUE TITLE
Admin API - Delete Admin Account

### ISSUE THREAD

**Issue by brylie** [NREL/api-umbrella#151]
Create a REST endpoint in the Admin API for deleting [Admin Accounts](http://apiumbrella.io/docs/admin-api/#admin-accounts).

User Story
---
```
As a developer extending API Umbrella Web
I need to delete Admin Accounts via the Admin REST API
so that I can ensure the Admin Users collections are consistent with my application
```

**Comment by brylie** [comment]
Possibly related to issue #139

**Comment by GUI** [comment]
Addressed in #152
